AdobeStockAsset/Test/_files/asset.php

```diff
@@ -82,11 +30,13 @@
         'data' => [
             'id' => 1,
             'is_licensed' => 1,
-            'category_id' => $categoryId,
-            'creator_id' => $creatorId,
-            'media_gallery_id' => $savedMeidaAsset->getId()
+            'category_id' => 42,
+            'creator_id' => 42,
+            'media_gallery_id' => $savedMediaAsset->getId()
```

Comment:
@coderimus $savedMediaAsset is `null` here during integration tests execution leading to fatal error

Reply:
@sivaschenko thank you for the review!

Ah, yes, this problem caused by the `require_once`. In 2 comments before this one, I explained why I user `require` instead of `require_once` but look like I didn't commit this change :( Now, changes added and tests should be fine. Let's wait for the result :)

![image](https://user-images.githubusercontent.com/5318512/82340163-912f7600-99f7-11ea-9342-d908e90c3a2a.png)